ng2-material/components/menu/menu.ts

```diff
@@ -0,0 +1,125 @@
+import {Component, ViewEncapsulation, ElementRef, Renderer} from "angular2/core";
+import {DOM} from "angular2/src/platform/dom/dom_adapter";
+
+
+@Component({
+  selector: 'md-menu',
+  template: `<ng-content></ng-content>`,  
+  encapsulation: ViewEncapsulation.None
+})
+export class MdMenu {
+  private _menuElement: HTMLElement;
+  private _triggerElement: Element;
+  private _menuContent: Element;
+  private _menuContainers: any;
+  private _isOpen: boolean = false;
+
+  constructor(private _element: ElementRef, private _renderer: Renderer) {
+    this._menuElement = this._element.nativeElement;
```

Comment:
why using a new var to store this ?
You have two choices: 
Keeping `_element` and remove `_menuElement`:
```
export class MdMenu {
  private _triggerElement: Element;
  [...]

  constructor(private _element: ElementRef [...]
```

or if you want to keep _menuElement as name, you can use the "magic declaration" of class property from constructor parameter:

```
export class MdMenu {
  private _triggerElement: Element;
  [...]

  constructor(private _menuElement: ElementRef [...]
```